modules/st2-auto-form/fields/string.js

```diff
@@ -18,7 +18,7 @@ export default class StringField extends BaseTextareaField {
   static icon = 'T'
 
   fromStateValue(v) {
-    return v !== '' ? v : void 0;
+    return v !== '' ? v.replace(/[&\\#()$~'":*?<>{}]/g, '') : void 0;
```

Comment:
Doesn't this stop me putting YAQL in, it's quite common to put YAQL in an input field. So the special characters are needed, as won't this stop me doing anything like saying <% ctx().field1 %> as an input, as you are stripped off <>.
Equally alot of those characters that are stripped out are yaql expressions, or regular expressions that I might be using in an expression.

Reply:
Simiarly { and } would be used to enter any jinja expression